public/app/plugins/datasource/prometheus/configuration/DataSourceHttpSettingsOverhaulPackage.tsx

```diff
@@ -134,6 +134,12 @@ export const DataSourcehttpSettingsOverhaul = (props: Props) => {
         urlTooltip={urlTooltip}
       />
       <hr className={`${styles.hrTopSpace} ${styles.hrBottomSpace}`} />
+      {sigV4Selected && (
+        <Alert title="Deprecation Notice" severity="warning">
+          The SigV4 authentication in the core Prometheus data source is deprecated. Please use the Prometheus Amazon
```

Comment:
I think based on the conversation in Slack we'll want this to say "Amazon Prometheus" instead.